cmd/commands/git-source.go

```diff
@@ -389,8 +551,11 @@ func RunGitSourceEdit(ctx context.Context, opts *GitSourceEditOptions) error {
 		return fmt.Errorf("failed to write the updated config.json of git-source: %s. Err: %w", opts.GsName, err)
 	}
 
-	log.G(ctx).Info("Pushing updated GitSource to the installation repo")
-	if err := apu.PushWithMessage(ctx, repo, fmt.Sprintf("Persisted an updated git-source '%s'", opts.GsName)); err != nil {
```

Comment:
I'm not sure why apu.PushWithMessage got included in this pr

Reply:
Becase you didn't handle conflicts well, please fix.